pkg/config/config.go

```diff
@@ -30,12 +30,12 @@ func New() Config {
 		OauthProxyContainerName: getEnv("OAUTH_PROXY_CONTAINER_NAME", "mdc-oauth-proxy"),
 
 		MDCImageStreamName:        getEnv("MDC_IMAGE_STREAM_NAME", "mdc-imagestream"),
-		MDCImageStreamTag:         getEnv("MDC_IMAGE_STREAM_TAG", "latest"),
+		MDCImageStreamTag:         getEnv("MDC_IMAGE_STREAM_TAG", "1.1"),
 		OauthProxyImageStreamName: getEnv("OAUTH_PROXY_IMAGE_STREAM_NAME", "mdc-oauth-proxy-imagestream"),
 		OauthProxyImageStreamTag:  getEnv("OAUTH_PROXY_IMAGE_STREAM_TAG", "latest"),
 
 		// these are used when the image stream does not exist and created for the first time by the operator
-		MDCImageStreamInitialImage:        getEnv("MDC_IMAGE_STREAM_INITIAL_IMAGE", "quay.io/aerogear/mobile-developer-console:latest"),
+		MDCImageStreamInitialImage:        getEnv("MDC_IMAGE_STREAM_INITIAL_IMAGE", "quay.io/aerogear/mobile-developer-console:1.1"),
```

Comment:
I added this additional tag (and others) dynamically through the quay.io web interface. We can improve on the automation to do what we want in future.